### PR TITLE
SREP-1186: Add stub for configure-alertmanager-operator

### PR DESCRIPTION
**Pull request by lisa**
Note that the hash length and template file need to make their way to
the operator's git repo (the favoured hash length is 8 characters, and
the template does not exist yet).

Signed-off-by: Lisa Seelye <lseelye@redhat.com>

### CODE REVIEW

File: configure-alertmanager-operator/configure-alertmanager-operator.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+services:
+- hash: none
+  name: configure-alertmanager-operator
+  path: /hack/olm-registry/olm-artifacts-template.yaml
```
Comment (jharrington22) [inline configure-alertmanager-operator/configure-alertmanager-operator.yaml]: We would want to change the name of this to sss-artifacts-template.yaml ?

**Comment by jmelis** [comment]
Also, don't forget to add a new context to https://github.com/app-sre/saas-hive/blob/master/config.yaml

**Comment by jharrington22** [comment]
My fault, I did not mention the context specifically to @lisa :)

File: configure-alertmanager-operator/configure-alertmanager-operator.yaml (hunk 1 above)
Comment (lisa) [reply configure-alertmanager-operator/configure-alertmanager-operator.yaml]: Maybe? I thought about it, but wanted to keep it consistent.

Comment (jmelis) [reply configure-alertmanager-operator/configure-alertmanager-operator.yaml]: fwiw we don't have a preference from the app-sre side of things

**Comment by lisa** [comment]
> Also, don't forget to add a new context to https://github.com/app-sre/saas-hive/blob/master/config.yaml

Done!

**Review by jharrington22 — APPROVED** [review]
LGTM - lets wait for @jmelis to merge when ready.

File: configure-alertmanager-operator-services/configure-alertmanager-operator.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+services:
+- hash: none
+  name: configure-alertmanager-operator
+  path: /hack/olm-registry/olm-artifacts-template.yaml
```
Comment (jmelis) [inline configure-alertmanager-operator-services/configure-alertmanager-operator.yaml]: this path doesn't exist yet, right?

Comment (lisa) [reply configure-alertmanager-operator-services/configure-alertmanager-operator.yaml]: Is that in https://github.com/openshift/configure-alertmanager-operator ?

Comment (lisa) [reply configure-alertmanager-operator-services/configure-alertmanager-operator.yaml]: It is, working on a PR.

Comment (lisa) [reply configure-alertmanager-operator-services/configure-alertmanager-operator.yaml]: @jmelis https://github.com/openshift/configure-alertmanager-operator/pull/16 was merged, can you have another look here?

**Comment by jharrington22** [comment]
LGTM - merged master to update branch

**Review by jharrington22 — APPROVED** [review]
Checks passed.

**Pull request merged**